indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -72,6 +72,14 @@ class AbstractReviewingState(RichIntEnum):
     mixed = 5
 
 
+class AbstractNotifiedStates(RichIntEnum):
```

Comment:
This is because we don't have default templates for withdrawal and duplicates, right?

imo, this could just be a set referencing the AbstractState enum values. This way there's no danger of them going out of sync:
```python
{AbstractState.submitted, AbstractState.accepted, ...}
```
I think keeping the titles as `Accepted` instead of `Accept` is fine, but if you want to use the latter then you can use a dictionary instead:

```python
{AbstractState.submitted: 'Accept', ...}
```

Reply:
Not because we don't have templates, but because we don't notify those actions. 

If withdrawal operation is supposed to be done by the user who submitted, there's no point on notifying them.
If duplicated, neither.

Reply:
I haven't checked duplicates but withdrawal does send notifications

Reply:
In that case I can include it and create a template for it if you want

Reply:
ok that'd be great!